eosdis-nasa/earthdata-pub-api
# Edpub 1315 ornl demo

# Description

Updates the ORNL Test Workflow to showcase EDPub Capabilities

## Spec

See Ticket:  https://bugs.earthdata.nasa.gov/browse/EDPUB-1315

Requires https://github.com/eosdis-nasa/earthdata-pub-api/pull/131 to be approved and merged first

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Create new ORNL request
4. Change the workflow of the new request to the ORNL Test Workflow
5. Verify that workflow diagram shows correct steps
